### ISSUE TITLE
Animgraph preview model

### ISSUE THREAD

**Issue by Kabubu2** [Facepunch/sbox-issues#3098]
### Describe the bug

I want to change/add preview model into an animgraph however the explorer can't go change folders/update

### To Reproduce

Choosing any of these folders or going up on the hierarchy doesn't do anything. The only way to force it to update is to close the preview and open it again.

### Expected behavior

It should update each time that I try to go into a folder

### Media/Files

![image](https://user-images.githubusercontent.com/18005111/230071570-dca14479-4dd9-48f2-a7d9-2c5659f3fd02.png)

**Comment by trende2001** [comment]
yeah it seems the animgraph asset browser is a bit ducked up, i've had the same issues
i had to quadruple click on the folders to go to the directory and refresh it so i can get the models i want to preview
